docs/semgrep-code/overview.md

```diff
@@ -0,0 +1,44 @@
+---
+slug: overview
+append_help_link: true
+title: Overview
+hide_title: false
+description: Learn about Semgrep Code, a static application security testing (SAST) tool |
+    designed to find security vulnerabilities in your first-party code.
+tags:
+  - Semgrep Code
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+import SemgrepScan from "/src/components/concept/_semgrep-scan.mdx"
+
+Semgrep Code is a static application security testing (SAST) tool that inspects your application for security vulnerabilities.
```

Comment:
```suggestion
Semgrep Code is a static application security testing (SAST) tool that detects security vulnerabilities in your first-party code.
```

Just avoiding the use of "application".

Reply:
I think only the first part of this comment (double newline removal) was committed?